convert-hf-to-gguf.py

```diff
@@ -2461,13 +2464,19 @@ def parse_args() -> argparse.Namespace:
         help="directory containing model file",
     )
     parser.add_argument("--use-temp-file", action="store_true", help="use the tempfile library while processing (helpful when running out of memory, process killed)")
+    parser.add_argument("--verbose",      action="store_true",    help="increase output verbosity")
 
     return parser.parse_args()
 
 
 def main() -> None:
     args = parse_args()
 
+    if args.verbose:
+        logging.basicConfig(level=logging.DEBUG)
+    else:
+        logging.basicConfig(level=logging.INFO)
```

Comment:
This and most of the other basicConfig calls are a good candidate for a conditional expression.

```suggestion
    logging.basicConfig(level=logging.DEBUG if args.verbose else logging.INFO)
```